code/src/App.js

```diff
@@ -1,12 +1,32 @@
-import React from 'react'
-import data from './data.json'
+import React from "react";
+import data from "./data.json";
+import { Album } from "./Album";
+import { Header } from "./Header";
```

Comment:
You don't need the brackets when you are importing only one thing
```suggestion
import Album from "./Album";
import Header from "./Header";
```